docs/reference/commandline/dockerd.md

```diff
@@ -62,7 +62,7 @@ Options:
       --iptables                              Enable addition of iptables rules (default true)
       --ipv6                                  Enable IPv6 networking
       --label value                           Set key=value labels to the daemon (default [])
-      --live-restore                          Enable live restore of docker when containers are still running
+      --live-restore                          Enable live restore of docker when containers are still running (This option is not supported on Windows)
```

Comment:
Perhaps we should use `(Linux only)` to keep it shorter. This usage output is taken from the command line output. On a Windows daemon, this flag is not actually printed / visible (https://github.com/docker/docker/blob/109c26bd7482280946e356b33f17f4d82112dff3/daemon/config_unix.go#L85), wondering if we should have two different examples

@mstanleyjones I know there were some discussions ongoing w.r.t. windows-specific docs, any idea what's best here?